Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageDeliveryStatusCheckmarkView.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view that displays message delivery receipts.
+open class ChatMessageDeliveryStatusCheckmarkView: _View, ThemeProvider {
+    /// The data this view component shows.
+    open var content: MessageDeliveryStatus? {
+        didSet { updateContentIfNeeded() }
+    }
+    
+    /// The image view showing read state of the message.
+    open private(set) lazy var imageView = UIImageView()
+        .withoutAutoresizingMaskConstraints
+    
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+        
+        imageView.contentMode = .scaleAspectFit
+    }
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        let size: CGFloat = 14
+        widthAnchor.pin(equalToConstant: size).isActive = true
+        heightAnchor.pin(equalToConstant: size).isActive = true
+        
+        embed(imageView)
+    }
+    
+    override open func updateContent() {
+        super.updateContent()
+        
+        imageView.image = content.flatMap {
+            switch $0 {
+            case .pending:
+                return appearance.images.messageDeliveryStatusSending
+            case .sent:
+                return appearance.images.messageDeliveryStatusSent
+            case .read:
+                return appearance.images.messageDeliveryStatusRead
+            case .failed:
+                return appearance.images.messageDeliveryStatusFailed
+            default:
+                return nil
+            }
+        }
```

Comment:
`MessageDeliveryStatus` type is declared in LLC, appearance with images is in UI SDK. 

I was thinking about private extension for `MessageDeliveryStatus` with `tintColor` but that would require a `tintColor` to be a func so that we can pass `appearance` so it turned out to be unnecessary complex thing.